core_impl/src/main/java/com/google/instrumentation/stats/StatsSerializer.java

```diff
@@ -88,18 +89,18 @@ static void serialize(StatsContextImpl context, OutputStream output) throws IOEx
 
   // Deserializes input to StatsContext based on the binary format standard.
   // The encoded tags are of the form: <version_id><encoded_tags>
-  static StatsContextImpl deserialize(InputStream input) throws IOException {
+  static StatsContextImpl deserialize(InputStream input) throws IOException, ParseException {
```

Comment:
I was thinking to have something like this:
byte[] serialize(StatsContext ctxt) {}
StatsContext deserialize(byte[]) {}